run-queue

```diff
@@ -58,6 +60,18 @@ def consume_webhook_queue(channel, amqp):
         if request['action'] in ['opened', 'synchronize']:
```

Comment:
Would you mind changing this to `action`, so that all `if`s use the same variable? (It's just aliased above). Also, can we please change this to `elif`? It doesn't make a practical difference, but it more clearly points out that these are non-overlapping alternatives.

Reply:
done